src/Symfony/Component/Notifier/Bridge/Smsapi/SmsapiTransport.php

```diff
@@ -84,6 +84,9 @@ protected function doSend(MessageInterface $message): SentMessage
             throw new TransportException(sprintf('Unable to send the SMS: "%s".', $content['message'] ?? 'unknown error'), $response);
         }
 
-        return new SentMessage($message, (string) $this);
+        $sentMessage = new SentMessage($message, (string) $this);
+        $sentMessage->setMessageId($content['list'][0]['id'] ?? '');
```

Comment:
Should we really overwrite a previously set message id if the result of this expression is the empty string?

Reply:
looks like I was wrong, an `SmsMessage` is not a `RawMessage` and thus a message id will not be set in the constructor of `SentMessage`